components/tests/ui/robot_setup.sh

```diff
@@ -45,6 +47,34 @@ echo "A2,Treatment,10" >> "$BULK_ANNOTATION_CSV"
 # Create file for upload as File Annotation
 echo "Robot test file annotations" > "$FILE_ANNOTATION"
 
+# Create a python script for setting posX and posY on wellsamples
+# Used below after importing a plate
+WELLSCRIPT="wsScript.py"
```

Comment:
It will be better to have the script in a separate file so it is easier to understand and potentially re-use if need be